Sources/Serveable.swift

```diff
@@ -0,0 +1,10 @@
+
+/// Serves your order to a local or a remote bar
+
+public protocol Serveable {
+
+    associatedtype S
+
+    func serve(order: Order, delivery: (Delivery<S>)->())
```

Comment:
But why not use terminology that everyone is familiar with?

![screenshot 2016-08-31 17 04 08](https://cloud.githubusercontent.com/assets/45648/18133969/f7079108-6f9c-11e6-9114-041f0c850581.png)